examples/custom_textures.rs

```diff
@@ -133,17 +129,11 @@ impl Lenna {
         let (width, height) = image.dimensions();
         let data = image.into_rgba8();
 
-        let memory_properties = unsafe {
-            vk_context
-                .instance()
-                .get_physical_device_memory_properties(vk_context.physical_device())
-        };
-
         let texture = Texture::from_rgba8(
-            vk_context.device(),
+            vk_context,
```

Comment:
you can remove the need to pass queue and command_pool since they can now be obtained from the passed vk_context.
unless you want to keep the flexibility to pass alternative queue and command_pool

Reply:
Indeed, thank you.

By the way sorry about your PR but it was so long ago I needed to write some code to wrap my head around it all and I got to the point where I had something that I was ok with.